src/filesys.h

```diff
@@ -29,11 +29,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define DIR_DELIM_CHAR '\\'
 #define FILESYS_CASE_INSENSITIVE 1
 #define PATH_DELIM ";"
+static const std::string bad_names[] = { "CON", "PRN", "AUX", "NUL", "COM1", "COM2", "COM3", "COM4", "COM5", "COM6", "COM7", "COM8", "COM9", "LPT1", "LPT2", "LPT3", "LPT4", "LPT5", "LPT6", "LPT7", "LPT8", "LPT9", "" };
```

Comment:
@nerzhul That's exactly what I meant by "separate instance". Sorry if that wasn't the correct term.